projects/rero/ng-core/src/lib/record/files/files.component.ts

```diff
@@ -507,7 +500,18 @@ export class RecordFilesComponent implements OnDestroy, OnInit {
         }
 
         this.files = files;
-      }),
+       }),
+       tap(() => {
+          // Check if a file can be added.
+          const canAdd$ = this._config.files.canAdd
+            ? this._config.files.canAdd()
+            : of({ can: false, message: '' });
+          this._subscriptions.add(
+            canAdd$.subscribe((result: ActionStatus) => {
+              this.canAdd = result;
+            })
```

Comment:
single instruction, could removed `{}` --> `canAdd$.subscribe((result: ActionStatus) => this.canAdd = result)`